rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,222 @@
+
+```
+$ rancher volume ls
+```
+
+If you created the `app-data` volume from the previous section, you may wonder why it wasn't listed. You can include `inactive` volumes with the `--all` or `-a` flag.
+
+```
+$ rancher volume ls --all
+```
+
+#### Delete a Volume
+
+You can delete a volume with the `rancher volume rm` command.
+
+```
+$ rancher volume rm app-data
+```
+
+#### Volume States
+
+Volumes have five distinct states: `inactive`, `activating`, `active`, `detached`, and `removed`.
+
+A newly-created volume is in `inactive` state until a container attempts to attach to it.
+
+When a container is created, attached volumes enter `activating` state. Once the container enters the `running` phase, its volumes enter `active` state. Containers attaching to an already `active` volume will not impact that volume's state.
+
+When all containers attached to a volume are deleted, the volume enters `detached` state.
+
+A deleted volume is in `removed` state. These volumes no longer exist on any Docker host and won't appear in when listing volumes. The volume "shadow" objects exist in Rancher API for audit purposes.
+
+### Volume Scopes
+
+There are two different volume scopes. The scope refers to the level at which the volume is managed by Rancher.
+
+#### Environment Scope
+
+An `environment scope`, or `external` volume may be shared by all containers in an environment. Rancher schedules containers appropriately based on the volume driver's access mode. A user doesn't need to be aware of the technical
+
+The practical benefit of this scope is the ability to easily share data between different software services/stacks whose lifecycles are independently managed. The user is given full control and must [manage the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#managing-volumes) themselves.
+
+##### Shared Example
+
+Rancher requires environment volumes to already exist prior to creating a service consuming it. Any installed volume driver can be used. We will use the `local` driver because it ships with the Docker daemon.
+
+[Create the volume]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#create-a-volume).
+
+```
+$ rancher volume create --driver local redis-data-external
+```
+
+Define the volume with `external: true` modifier to indicate the volume's lifecycle is managed externally (environment scope). Save the `docker-compose.yml` file below to a directory.
+
+```yaml
+version: '2'
+services:
+  redis:
+    image: redis:3.0.7
+    volumes:
+    - redis-data-external:/data
+volumes:
+  redis-data-external:
+    driver: local
+    external: true
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+[List the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) to verify that `redis-data` volume is `active`.
+
+
+> **Note:** Scaling a service up or down mounts or unmounts the same shared volume.
+
+##### Per-Container Example
```

Comment:
This section doesnt make sense to me. It might just be its placement in the article or the fact that the heading "Per-Container" is used without explanation.